theme/views.py

```diff
@@ -40,11 +39,15 @@
 from theme.models import UserProfile
 from theme.utils import get_quota_message
 from .forms import SignupForm
-
+from django.template.defaulttags import register
 
 class UserProfileView(TemplateView):
     template_name='accounts/profile.html'
 
+    # @register.filter
```

Comment:
If the "End of Resource Listing" message is sufficient, the `filter` code here will be removed.